### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by LukasOldenburg** [MouseLand/cellpose#1028]
Hi! Great model and great results, thank you!

**Describe the bug**
I'm running the cyto3 model and want to batch process my images for faster inference.

What I do is:

model = models.Cellpose(gpu=True, model_type='cyto3')
images = [io.imread(x) for x in paths_of_2D_images]
masks, _, _, _ = model.eval(images, channels=[0, 0], batch_size=batch_size, diameter=None, z_axis=0)

images is a list of 32 2D images.
len(images) = 32
images[0].shape = (2048, 2048)

I want to control the amount of parallel processed images by the batch size parameter. E.g. batch_size=32

Checking the logs, it seems like the images are processed sequentially, although the GPU memory goes up for higher batch sizes. 

Could you clarify how to actually run batched inference for this specific case?
(Also referring to  [Increasing Cellpose throughput #447](https://github.com/MouseLand/cellpose/issues/447) )

Thank you,
Lukas

**Comment by carsen-stringer** [comment]
yes if you input a list, then they run one at a time. to get the batches to include tiles across images, please feed them in as an array (as long as they are all the same size). we do not currently support batches with tiles from multiple images if they are different sizes. (please also upgrade to the github version of cellpose to make sure this is working well)

added some info in [docs/inputs.rst](docs/inputs.rst). I'll close this for now but will reopen if it's not working
